html/dom/dynamic-markup-insertion/opening-the-input-stream/event-listeners.window.js

```diff
@@ -0,0 +1,42 @@
+test(t => {
+  const frame = document.body.appendChild(document.createElement("iframe")),
+        body = frame.contentDocument.body;
+  t.add_cleanup(() => frame.remove());
+  frame.contentDocument.addEventListener("x", t.unreached_func());
+  body.addEventListener("x", t.unreached_func());
+  frame.contentDocument.open();
+  frame.contentDocument.close();
```

Comment:
The "close()" here seems not great since if someone implemented close() as removing the event listeners the test would still pass.